docs/run-platform-server/monitoring.mdx

```diff
@@ -7,11 +7,45 @@ import { CodeExample } from "@site/src/components/CodeExample";
+
+One real world example for exporting from a bare metal horizon installation(horizon has been installed directly onto an operating system), use[FluentBit and the Prometheus Exporter](https://docs.fluentbit.io/manual/pipeline/outputs/prometheus-exporter) on same host machine as Horizon is running. FluentBit will perform a simple port forwarding pipeline on host machine. Configure the input to be Horizon's `localhost:<ADMIN_PORT>\metrics` and the output to be the `host` and `port` representing the target network interface and port on the host machine, and then configure your monitoring infrastructure to scrape that host address.
+
+In container orchestrated environents such as kubernetes, you will use the same exporter strategy, and have already setup metrics infrastructure deployment on cluster such as Prometheus and Grafana via the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator), and will just need to configure that infrastructure to scrape the horizon pod based on the ADMIN_PORT.
+
+### Data model
+
+There are numerous application metrics keys emitted by horizon at runtime, encoded in all four types of exposition formats: `counter`, `gauge`, `historgram`, `summary`. Each key is further qualified with labels for more granularity. To summarize, we can highlight the groupings of metrics keys by common function denoted in the prefix of their name:
+
+`go_` - golang specfici runtime performance `horizon_txsub_` - attributes of horizon transaction submission sub system if enabled. `horizon_stellar_core_` - runtime attributes of stellar network reported by the captive core or a remote stellar core which horizon is configured. `horizon_order_book_` - runtime attributes of the in memory order book maintained by horizon of the current stellar network `horizon_log_` - counters of how many log messages printed at each severity level `horizon_ingest_` - performance measurements and stateful aspects of horizon's internal ingestion sub system `horizon_http_` - statistics and measurements of horizon's HTTP API service, all aspects of request/response load and timings. `horizon_history_` - statistics on horizon ingested historical ledgers `horizon_db_` `process_` - generic host machine compute measurements
+
+And for each key, there will be possibility of 0 or more labels, the serialized output(exposition) format follows this template:
+
+```
+<metrics_key_name>{label_1="value",label_2="value",,,} <value>
+```
+
+Rather than listing all individual metrics keys in docs, as they change often, the recommendation is to perform an HTTP GET against the horizon metrics endpoint,`localhost:<ADMIN_PORT>\metrics`, using any http client(browser, curl, wget, etc) and the response will have the metrics keys and additional meta information on each metric key for description and type(counter, guage, histogram, summary), as an example for one key, `horizon_http_requests_duration_seconds`:
```

Comment:
```suggestion
Rather than listing all individual metrics keys in docs, as they change often, the recommendation is to perform an HTTP GET against the horizon metrics endpoint,`localhost:<ADMIN_PORT>/metrics`, using any http client(browser, curl, wget, etc) and the response will have the metrics keys and additional meta information on each metric key for description and type(counter, guage, histogram, summary), as an example for one key, `horizon_http_requests_duration_seconds`:
```

Reply:
there's no difference in the suggestion?